library/kubevirt_ovm.py

```diff
@@ -1,4 +1,8 @@
 #!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2018, Karim Boumedhel <@karmab>, Irina Gulina <@alexxa>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 
 from ansible.module_utils.basic import AnsibleModule
```

Comment:
According to [Ansible's guidelines](http://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html), there should be nothing between copyright and `ANSIBLE_METADATA`.